### ISSUE TITLE
Logo in Chrome tray with count of active tests on the page

### ISSUE THREAD

**Issue by kingo55** [mint-metrics/mojito-chrome-inspector#3]
We could show how many active tests are on a page so you can tell what's running at a glance.

It would look just like how the FB pixel helper does it:

<img width="34" alt="Screen Shot 2020-02-05 at 09 09 42" src="https://user-images.githubusercontent.com/2361388/73792028-96c1d100-47f7-11ea-89a2-3477eead09b6.png">

**Comment by dapperdrop** [comment]
Yep I like this idea. It will reinforce the visuals of the active state as well.
